tests/kerascv/layers/ssd_box_coder_test.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from kerascv.layers.ssd_box_coder import SSDBoxCoder
+
+
+def test_encode_decode_variance():
```

Comment:
Do we want to set all modes (e.g. graph, eager) of keras?

Reply:
Is that testing util exposed?

Reply:
Do you mean like in https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/test_utils.py?